connectors/embeddings-vector-database/element-templates/versioned/embeddings-vector-database-outbound-connector-1.json

```diff
@@ -17,7 +20,7 @@
     "value" : "bpmn:ServiceTask"
   },
   "engines" : {
-    "camunda" : "^8.2"
+    "camunda" : "^8.8"
```

Comment:
Is it OK to modify the engine version?

Reply:
I think so, `8.2` was definitely wrong, so I would not see a good use case to keep it.